### ISSUE TITLE
Forum: Homepage revise favourite and delete

### ISSUE THREAD

**Issue by kathreenriel** [BCcampus/eypd#574]
The home page of the Forum has two options in the activity feed which are confusing for the user especially the delete function.  

Take out the favourite and delete options. So that both Learners and Organizers cannot favourite or delete activities/announcements on this home page. 

[Test Forum Link](https://earlyyearsbc.ca/groups/test-appetite-to-play/forum/)
<img width="792" alt="export and delete" src="https://user-images.githubusercontent.com/16693738/48864385-40e34380-ed81-11e8-9c16-74d41fed0043.png">

**Comment by mandily-p** [comment]
I don't have the right permission level to access buddypress forum settings. @kathreenriel are you able to give them to me?

**Comment by kathreenriel** [comment]
@mandily-p I have the same level of access as you. Maybe you need Keymaster?

**Comment by bdolor** [comment]
requires discussion between peeps

**Comment by paulagaube** [comment]
@mandily-p @kathreenriel Yes, we need Keymaster to administer the forums. I found this description online: 
The keymaster role is not a default WordPress role, rather it is part of buddypress. Once you install buddypress the keymaster role is created and is to bbpress what the Administrator role is to WordPress. The keymaster controls the forums.
I scooted over to Brad's desk and he made me a Keymaster in EYPD CERT.  I have added the Keymaster role to your Administrator account in CERT too....
